website/blog/2023-03-24-databend-weekly-86.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "This Week in Databend #86"
+date: 2023-03-24
+slug: 2023-03-24-databend-weekly
+tags: [databend, weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: andylokandy
+  - name: ariesdevil
+  - name: b41sh
+  - name: BohuTANG
+  - name: Chasen-Zhang
+  - name: dantengsky
+  - name: everpcpc
+  - name: leiysky
+  - name: lichuang
+  - name: mergify[bot]
+  - name: PsiACE
+  - name: RinChanNOWWW
+  - name: SkyFan2002
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: wubx
+  - name: xinlifoobar
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: yufan022
+  - name: zhang2014
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### FlightSQL Handler in Progress
+
+Flight SQL is an innovative open database protocol that caters to modern architectures. It boasts a columnar-oriented design and provides seamless support for parallel processing of data partitions.
+
+The benefits of supporting FlightSQL include reducing serialization and deserialization during query execution, as well as easily supporting SDKs in different languages using predefined proto.
+
+Databend is actively working on supporting the FlightSQL Handler. If you're interested, visit the following materials.
```

Comment:
We're currently engaged in developing support for the FlightSQL Handler. If you're interested, refer to the following links: